pkg/contexts/credentials/builtin/mvn/identity/identity.go

```diff
@@ -0,0 +1,76 @@
+package identity
+
+import (
+	"net/http"
+	"path"
+
+	. "net/url"
+
+	"github.com/open-component-model/ocm/pkg/common"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials/cpi"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials/identity/hostpath"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi/accspeccpi"
+	"github.com/open-component-model/ocm/pkg/listformat"
+	"github.com/open-component-model/ocm/pkg/logging"
+)
+
+const (
+	// CONSUMER_TYPE is the mvn repository type.
+	CONSUMER_TYPE = "Repository.maven.apache.org"
```

Comment:
True, I totally agree. Let's capture some proposals...

A | B | C | D
---|---|---|---
oci|OCIRegistry|OciCredentialMatcher| ?
helm|HelmRepository|HelmCredentialMatcher| ?
s3|SimpleStorageService|S3CredentialMatcher| ?
maven|MavenRepository|MavenCredentialMatcher| ?
npm|NpmRegistry|NpmCredentialMatcher| ?
sign|Signingserver.gardener.cloud|GardenerSigningCredentialMatcher| ?
build|Buildcredentials.ocm.software|GardenerConfigCredentialMatcher| ?
git|Github|GithubCredentialMatcher| ?
vault|HashiCorpVault|VaultCredentialMatcher| ?
wget|Wget|WgetCredentialMatcher| ?

Currently available and listed: https://ocm.software/docs/cli-reference/help/credential-handling/

@morrison-sap maybe we should ask our stakeholders for input?

Reply:
Having the domain part of the type name should only be required in case the type is not for something common, e.g. the ones for sign/build (by the way: the signing server will move under the ocm org somewhen, then maybe gardener.cloud is not a good idea as name). For maven and npm I would go for MavenRepository and NPMRegistry.

@hilmarf wrt to feedback from stakeholders, yes, why not? If we want to invite others for feedback in this issue, we should have a clear question and task for reviewers. Can you please bring this or a follow-up comment into shape that can act as basis for gathering feedback? Once ready I'll add colleagues (looking over the issues we got in the last year).